x-pack/test/cases_api_integration/security_and_spaces/tests/common/cases/resolve_case.ts

```diff
@@ -50,7 +50,7 @@ export default ({ getService }: FtrProviderContext): void => {
     });
 
     it('should resolve a case with no comments', async () => {
-      const postedCase = await createCase(supertest, getPostCaseRequest());
+      const postedCase = await createCase(supertest, postCaseReq);
```

Comment:
I assume that because the `getPostCaseRequest()` has `customFields` failed. I think we should `getPostCaseRequest` and not add the `customFields` in `getPostCaseRequest`. See my previous comment.

Reply:
Actually here I am not sure what was wrong because the test passed in local and failed in pipeline